polars/polars-lazy/src/logical_plan/builder.rs

```diff
@@ -127,7 +127,10 @@ impl LogicalPlanBuilder {
         let path = path.into();
         let mut file = std::fs::File::open(&path)?;
         let mut magic_nr = [0u8; 2];
-        file.read_exact(&mut magic_nr)?;
+        match file.read_exact(&mut magic_nr) {
```

Comment:
Maybe we can do a `map_err` + `?` combination?.

Reply:
Oh, you are right!

Done 😄